dintero-checkout-for-woocommerce.php

```diff
@@ -48,6 +51,19 @@ class Dintero {
 		 */
 		public $order_management;
 
+		/**
+		 * The reference the *Singleton* instance of this class.
+		 *
+		 * @var Dintero $instance
+		 */
+
```

Comment:
I dont see a class property set for this comment, was it added and the property forgotten?